doc/nrf/app_dev/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_smp.rst

```diff
@@ -0,0 +1,460 @@
+.. _ug_nrf54h20_suit_smp:
+
+SUIT and SMP protocol (including SUIT SMP extension)
+####################################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+SUIT DFU offers a flexible and extensible update process.
+You can use the Zephyr management subsystem, via the SMP protocol, to manage SUIT-based images alongside existing MCUBoot images.
+
+Slots and Images in Zephyr
+==========================
+
+In the nRF Connect SDK, *slot* and *image* concepts are derived from MCUboot.
+An *image* consists of two slots: a *primary* and a *secondary*.
+The application runs from the primary slot, while updates are uploaded to the secondary slot.
+MCUboot is responsible for swapping these slots during boot.
+
+The slots support a single upgradable application, with equal sizes.
+Zephyr supports up to two images, each represented by an index.
+
+.. note::
+   Image metadata, such as version information, is embedded within the image payload.
+
+Components in SUIT
+==================
+
+SUIT defines a *component* as an updatable logical block, which can be firmware, software, configuration, or data.
+This is similar to how MCUmgr defines an *image*.
+Updates are delivered to the device as a SUIT *envelope,* containing SUIT manifests and, optionally, integrated payloads.
+The relationships between components are maintained through SUIT manifests.
+
+Metadata, such a version information, expected image digest, image size is embedded into SUIT manifest.
+
+MCUmgr and SUIT - Key Similarities and Differences
+===================================================
+
+The following table summarizes key similarities and differences between MCUmgr and SUIT:
+
++-----------------------------+-------------------------+----------------------------------+
+| Feature                     | MCUmgr                  | SUIT                             |
++=============================+=========================+==================================+
+| Metadata                    | Attached to the images  | Defined by a separate SUIT       |
+|                             |                         | manifest                         |
++-----------------------------+-------------------------+----------------------------------+
+| Image Topology              | Slots (primary,         | Tree structure with manifests    |
```

Comment:
**Tree** structure with manifests grouped into a hierarchical **tree**

Reply:
Could you describe this feature in a more verbose way? Would be helpful to edit this.